Makefile

```diff
@@ -98,6 +98,9 @@ $(NATIVE_DLL): $(SQLITE_OUT)/$(LIBNAME)
 	@mkdir -p $(NATIVE_TARGET_DIR)
 	cp $< $(NATIVE_TARGET_DIR)/$(LIBNAME)
 
+android-arm: $(SQLITE_UNPACKED) jni-header
+	./docker/dockcross-android bash -c 'make clean-native native CROSS_PREFIX=/usr/arm-linux-androideabi/bin/arm-linux-androideabi- OS_NAME=Android OS_ARCH=arm'
```

Comment:
A concern here is OS_NAME (Android) should match JVM system property `os.name` of DalvikVM for Android.